static/app/components/searchQueryBuilder/tokens/useSortedFilterKeyItems.tsx

```diff
@@ -170,11 +208,22 @@ export function useSortedFilterKeyItems({
       return [
         ...searchKeyItems,
         ...getFilterSearchValues(flatKeys, {getFieldDefinition}),
+        // only show the logic items if not the first item, as they're not allowed to be the first item
```

Comment:
Same comment as on the other PR, IMO it would be better if we didn't have this check